config/routes.rb

```diff
@@ -35,4 +35,8 @@
       post 'highlight', format: /json/
     end
   end
+
+  namespace :admin do
```

Comment:
Is there any reason for nesting this inside of the `admin` namespace? Why not simply `/settings`.

Reply:
I sticked to the documentation of https://github.com/huacnlee/rails-settings-cached here. Similar to `telegram/webhook_controller.rb` which could be `telegram_controller.rb` as well.